docs/settings/triggers/snowboy.md

```diff
@@ -4,23 +4,44 @@ You can create your magic word by connecting to [Snowboy](https://snowboy.kitt.a
 
 Once downloaded, place the file in your personal config folder and configure snowboy in your [your settings](../settings.md) following the table bellow
 
-| parameter   | required | type   | default | choices         | comment                                                                                          |
-|-------------|----------|--------|---------|-----------------|--------------------------------------------------------------------------------------------------|
-| pmdl_file   | TRUE     | string |         |                 | Path to the snowboy model file. The path can be absolute or relative to the brain file           |
-| sensitivity | FALSE    | string | 0.5     | between 0 and 1 | Increasing the sensitivity value lead to better detection rate, but also higher false alarm rate |
+| parameter      | required | type   | default | choices         | comment                                                                                          |
+|----------------|----------|--------|---------|-----------------|--------------------------------------------------------------------------------------------------|
+| keyword_file   | TRUE     | string |         |                 | Path to the snowboy model file. The path can be absolute or relative to the brain file           |
+| keywords       | FALSE    | string |         |                 | If you want to use multiple keywords                                                             |
+| sensitivity    | FALSE    | string | 0.5     | between 0 and 1 | Increasing the sensitivity value lead to better detection rate, but also higher false alarm rate |
+| apply_frontend | FALSE    | string | False   | True/False      | Some universal models (umdl files) need apply_frontend to work properly                          |        
 
-## Example settings
+## Example settings for single keyword
 
 ```yaml
-# This is the trigger engine that will catch your magic work to wake up Kalliope. Only Snowboy is available so far
+# This is the trigger engine that will catch your magic work to wake up Kalliope.
 default_trigger: "snowboy"
 
 # Trigger engine configuration
 triggers:
   - snowboy:
-      pmdl_file: "trigger/snowboy/resources/kalliope-FR-40samples.pmdl"
+      keyword_file: "trigger/kalliope-FR.pmdl"
 ```
 
+## Note
+Some universal models contains two keywords, if we want to set a custom sensitivity we need to add two sensitivities for each keyword.  
+
+## Example settings for multiple keywords, where we set the sensitivity for jarvis.umdl and set apply_frontend to True
+
+```yaml
+# This is the trigger that will catch your magic work to wake up Kalliope
+default_trigger: "snowboy"
+triggers:
+  - snowboy:
+      apply_frontend: True
+      keywords:
+        - keyword_file: "trigger/kalliope-FR.pmdl"
```

Comment:
Not really a nice presentation.
I would go with
```
triggers:
  - snowboy:
      apply_frontend: True
      keywords:
        - file_path: "trigger/kalliope-FR.pmdl"
          sensitivity: 0.7
        - file_path: "trigger/jarvis.pmdl"
          sensitivity: 0.6
```

Reply:
I changed it to file_path